### PR TITLE
WIP - Add Labels (using Hashtag names from Gerrty)

### PR DESCRIPTION
**Pull request by EmilienM**
This patch adds supports for labels in the changes, but it's limited
now (only view labels).
Also, we might need to rename the methods since Hashtag naming was good
for Gerrit but not for Github.

### CODE REVIEW

**Comment by EmilienM** [comment]
wow

**Review by mandre — COMMENTED** [review]
Do you want to keep working on the labels feature in this branch, or do you instead want to merge as we go?

Things I'd like to see before calling this feature complete:
- [ ] retrieve possible labels for the project from github and store them in base
- [ ] add label should present a checkbox to the user with a list of possible labels
- [ ] possibly re-use the same color to display the label
- [ ] remove references to hashtags everywhere from the code and replace with labels. We'll need to rename the DB table. This will be a bit tricky because there's already a label table used for approvals I think.

**Comment by EmilienM** [comment]
> Do you want to keep working on the labels feature in this branch, or do you instead want to merge as we go?

please don't merge, it's highly WIP now.

File: hubtty/alembic/versions/399c4b3dcc9a_add_hashtags.py [hunk 1]
```diff
@@ -23,6 +23,11 @@ def upgrade():
     sa.Column('key', sa.Integer(), nullable=False),
     sa.Column('change_key', sa.Integer(), sa.ForeignKey('change.key'), index=True),
     sa.Column('name', sa.String(length=255), index=True, nullable=False),
+    sa.Column('description', sa.String(length=255), index=True, nullable=False),
```
[the remark this answers is not in the document]
Comment (EmilienM) [reply hubtty/alembic/versions/399c4b3dcc9a_add_hashtags.py:26]: yeah i'll revert that change

File: hubtty/sync.py [hunk 2]
```diff
@@ -1058,7 +1058,7 @@ def run(self, sync):
             data = dict(add=add, remove=remove)
             change.pending_hashtags = False
             # Inside db session for rollback
-            sync.post('changes/%s/hashtags' % (change.id,),
+            sync.post('repos/%s/labels' % (change.id,),
```
Comment (mandre) [inline hubtty/sync.py:1061]: This will have to be `change_id`.

**Review by mandre — CHANGES_REQUESTED** [review]
First review from hubtty \o/

**Review by mandre — COMMENTED** [review]
Testing review on own repo

**Comment by mandre** [comment]
Closed in favor of #39

**Pull request closed without merging**